src/arkode/arkode_lsrkstep.c

```diff
@@ -1305,6 +1306,7 @@ int lsrkStep_TakeStepSSPs3(ARKodeMem ark_mem, sunrealtype* dsmPtr, int* nflagPtr
     step_mem->nfe++;
     if (retval < 0) { return ARK_RHSFUNC_FAIL; }
     if (retval > 0) { return RHSFUNC_RECVR; }
+    ark_mem->fn_is_current = SUNFALSE;
```

Comment:
This works around clobbering `fn` but adds an extra RHS evaluation at the start of a step re-attempt and after every successful step if the Hermite interpolation model is used with an SSP method. Instead of setting `fn_is_current` to false, `vtemp3` could be use the store the RHS values as it's already allocated and unused.

Reply:
I believe we have agreed that users who prefer extra vectors over extra RHS evaluations should use RK tables instead and keep this module as low storage.

Reply:
`tempv3` is already allocated this adds no additional storage